academic-services/project-academic-knowledge/reference-affiliation-entity-attributes.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Affiliation entity attributes
```

Comment:
Should we refer to **https://review.docs.microsoft.com/en-us/academic-services/knowledge-exploration-service/reference-makes-api-entity-schema?branch=DarrinEide%2Fproject-academic-knowledge-migration** for entity schema so we don't have to maintain this in two places?

Reply:
That's my eventual goal, but that can wait till post-migration

---
In reply to: [383562916](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/119#discussion_r383562916) [](ancestors = 383562916)